src/main/scala/beam/utils/csv/readers/BeamCsvScenarioReader.scala

```diff
@@ -75,6 +76,7 @@ object BeamCsvScenarioReader extends BeamScenarioReader with ExponentialLazyLogg
     val linkIds =
       Option(rec.get("legRouteLinks")).map(_.split(ArrayItemSeparator).map(_.trim)).getOrElse(Array.empty[String])
     PlanElement(
+      tripId = rec.get("trip_id"),
```

Comment:
The value "trip_id" might be missing in the input plans.
I suggest using getIfNotNull or something similar